docs/plugins/spotify-connect.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Spotify Connect Plugin
+description: Features and Notes for the Spotify Connect Plugin
+---
+
+# Spotify Connect ![Preview image](../assets/icons/spotify-connect-icon.png){ align=right }
+
+Music Assistant has the ability to add [Spotify Connect](https://connect.spotify.com/) support to any MA player.
+
```

Comment:
Note that this provider is still in an early stage of development. Functionality is limited and bugs may occur.

Reply:
done